pages/home/home.html

```diff
@@ -1,21 +1,24 @@
+<!DOCTYPE html>
 <section id="main-content">
   <md-card layout-fill style="position: relative;">
     <div id="home-page-welcome-image-container" flex layout="column" layout-fill class="oppia-foundation-clearfix">
-      <img ng-src="https://placekitten.com/3000/1200" class="home-page-welcome-image">
-      <md-card-content layout="column" layout-align="center center" class="home-page-welcome-image-overlay horizontal-padding">
-        <div class="home-page-welcome-text-overlay">
-          <h1 hide-xs hide-sm class="oppia-foundation-horizontal-padding">
-            We're building a global classroom
-          </h1>
+      <img src="/static/images/cheering-child.png" class="home-page-welcome-image">
+      <md-card-content layout="column" layout-align="center center" class="home-page-welcome-image-overlay horizontal-padding oppia-foundation-green-background">
+        <div class="home-page-welcome-text-overlay home-page-welcome-text-box">
+          <md-card-title class="home-page-body-card-title">
+            <h1 hide-xs hide-sm class="oppia-foundation-horizontal-padding md-display-2">
+              We're building a global classroom
+            </h1>
+            <p hide-gt-sm style="font-size: 14pt; font-weight: bold;">
+              We're building a global classroom
+            </p>
+          </md-card-title>
           <p></p>
-          <h3 hide-xs hide-sm class="oppia-foundation-horizontal-padding">
+          <h3 hide-xs hide-sm class="oppia-foundation-horizontal-padding md-title">
             Students from every corner of the world deserve access to engaging, personalized education.
           </h3>
-          <p hide-gt-sm style="font-size: 10pt; font-weight: bold;">
-            We're building a global classroom
-          </p>
           <p hide-gt-sm style="font-size: 7pt;">
-            Students from every corner of the world deserve access to engaging, personalized education.
+              Students from every corner of the world deserve access to engaging, personalized education.
```

Comment:
Deindent by 2

Reply:
Done! I have disabled my VSCode's auto-indent settings to stop this from happening.